### ISSUE TITLE
Core repo config validation needs to be removed or improved

### ISSUE THREAD

**Issue by dvstans** [ORNL/DataFed#856]
There are some very basic checks that are run against repo configuration loaded from the DB. These checks are only good at detecting very basic config mistakes AND the errors are only reported to the log (no way for an admin to know about or view errors). It is probably better just to not validate these settings, then let the system fail when the repo is first used. This might produce an error that would lead admins to check the configuration. Alternatively, the system could validate the settings by pinging the repo servers when the settings are first entered or later changed.

**Comment by JoshuaSBrown** [comment]
Yes, I believe in hard failures on startup for things likes like this. I really dislike quiet failures for configuration issues that should be resolved on start.
